### PR TITLE
noz.de

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/193777 + [video](https://github.com/AdguardTeam/AdguardFilters/issues/193777#issuecomment-2507572485).

### CODE REVIEW

File: Filters/exceptions.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdguardFilters/issues/193777#issuecomment-2507572485
```
Comment (AdamWr) [inline Filters/exceptions.txt:1]: Is `#issuecomment-2507572485` needed here, or perhaps it can be removed?

**Review by AdamWr — APPROVED** [review]

File: Filters/exceptions.txt [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
+@@||t.cpdsrv.de^|
```
Comment (Alex-302) [inline Filters/exceptions.txt:2]: As I see, on sites used only `t.cpdsrv.de` domain. Better exclude `||cpdsrv.de^`

File: Filters/exceptions.txt (hunk 1 above)
Comment (zloyden) [reply Filters/exceptions.txt:1]: Just pointed right to the video.

Comment (zloyden) [reply Filters/exceptions.txt]: I think it's ok to leave just the original link as you said earlier, so I've done it.

**Review by AdamWr — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**